client/driver/docker.go

```diff
@@ -183,6 +183,20 @@ func (d *DockerDriver) createContainer(ctx *ExecContext, task *structs.Task) (do
 	}
 	hostConfig.NetworkMode = mode
 
+	// Handle the priviledged flag
+	priviledged, ok := task.Config["priviledged"]
+	if !ok || priviledged == "" {
+		d.logger.Printf("[WARN] driver.docker: priviledged flag not set, defaulting to non-priviledged")
```

Comment:
I thought this would be a nice warning in case it was an operator error, since we are defaulting to a value. Also, a monitoring system might be looking at the messages.